### PR TITLE
API Launch Blog Announcement

### PR DESCRIPTION
**Pull request by lforalaska**

### CODE REVIEW

File: src/posts/helloworldapi.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+## Getting your API key
+
+1. Fill out this [form](https://signup.contenda.co)
+2. Check your email for the key!
+
+![UI screenshot of the signup page](/blogimages/signup_branded.png)
+
+## Making an identity call
+
+Now that you have your shiny new API call, you're going to immediately trade it for a shinier access token! Don't worry though, your API key entitles you to infinite access token trades.
```
Comment (GradySalzman) [inline src/posts/helloworldapi.md:27]: I think `new API call` be `new API Key` ?

File: src/posts/helloworldapi.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+For your first job, you only need 2 things: 
+
+1. A video that's currently hosted online (`source_id`)
+2. Decide if your output should be an overview blog or a deep technical tutorial (`type`)
+
+Here's the simplified request: 
+
+```bash
+curl -X 'POST' \
+  'https://prod.contenda.io/api/v2/jobs/video-to-blog?token=asdfsadf' \
```
Comment (jlin99) [inline src/posts/helloworldapi.md:46]: can we make this
`https://prod.contenda.io/api/v2/jobs/video-to-blog?token=(your token here)`
its kinda easy to glaze over and just copy paste blindly i think

File: src/posts/helloworldapi.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+![Code screenshot of an identity cURL command](/blogimages/identityapi.png)
+
+Make sure you put in the email you'd like notifications from and your API key into the empty string placeholders!
+
+## Submitting your first video-to-blog job
+
+For your first job, you only need 2 things: 
+
+1. A video that's currently hosted online (`source_id`)
+2. Decide if your output should be an overview blog or a deep technical tutorial (`type`)
```
Comment (jlin99) [inline src/posts/helloworldapi.md:40]: can we make this clearer and do something like

1. `source_id`: A video that's currently hosted online
2. `type`: Decide if your output should be an overview blog (`presentation`) or a deep technical tutorial (`tutorial`)

**Review by jlin99 — APPROVED** [review]
smol suggestions for clarity but lgtm otherwise

**Pull request merged**